crowbar_framework/app/helpers/barclamp/ceph_helper.rb

```diff
@@ -35,5 +35,17 @@ def diskmodes_for_ceph(selected)
         selected.to_s
       )
     end
+    def replicas_number_for_ceph(selected)
+      options_for_select(
+        [
+          ["Automatically","0"],
+          ["One","1"],
+          ["Two","2"],
+          ["Three","3"],
+          ["Four","4"]
```

Comment:
IMHO a number field with min and max values would be much better. With that it is possible that the value gets submitted correctly as an integer and can be really validated as an integer.